travis.php.ini

```
@@ -0,0 +1,4 @@
+; PHP 8.1 make the Serializable Interface deprecated and it will be removed in
+; PHP 9, before migrating to PHP 9, we have to replace the interface with
+; serializing magic methods and remove this.
```

⛏️ A todo marker would be helpful to find this here in the future 🙂 
```suggestion
; TODO: PHP 9, before migrating to PHP 9, we have to replace the interface with
; serializing magic methods and remove this.
```